src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/entities/DefaultInstructMessage.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities;
+
+public class DefaultInstructMessage extends HomeOfficeInstruct {
```

Comment:
I think this class is redundant as it is not different to the base class. Could you remove and use the HomeOfficeInstruct directly?

Reply:
Removed DefaultInstructMessage and used HomeOfficeInstruct